html/yogatherapy.html

```diff
@@ -133,10 +149,8 @@ <h1>Virabhadrasana</h1>
               case of frozen shoulders. Know the steps of doing the posture, all
               its benefits, and contraindications by cicking below.
             </p>
-            <a
-              href="https://www.artofliving.org/in-en/yoga/yoga-poses/warrior-pose-virbhadrasana"
-              ><button>Read More</button></a
-            >
+            <a href="https://www.artofliving.org/in-en/yoga/yoga-poses/warrior-pose-virbhadrasana"><button>Read
```

Comment:
If you are using a tag, There is no need use **_button_** inside of it.
You can design it to look like a button.

``` diff
- <a href=""><button></button></a>
+ <a href=""></a>
```

> **Note**
> For styling a tag, If you want to add `padding` on it, set the display to equal to `inline-block`.

>**Warning**
> If you don't use `inline-block`, The `padding-top` and `padding-bottom` is not going to work.